.env.sample

```diff
@@ -1,7 +1,7 @@
-DATABASE_URL=postgres://postgres:@localhost/library_development?sslmode=disable
+DATABASE_URL=postgres://odoo-13:@localhost/library_development?sslmode=disable
 SESSION_SECRET=
 GOOGLE_CLIENT_ID=
 GOOGLE_CLIENT_SECRET=
 BACKUPS_PATH=/path/to/backups
 BACKUPS_LIMIT=30
-DOMAIN=http://localhost:3000
+DOMAIN=http://127.0.0.1:3000
```

Comment:
changes to .env.sample shouldn't be commited, .sample file is meant to be copied to .env and changed on the machine for development.